package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "lab-01-modular-patterns",
+  "version": "0.0.1",
+  "description": "lab-gnoevil",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "lint": "./node_modules/eslint/bin/eslint.js .",
+    "test": "./node_modules/mocha/bin/mocha.js"
```

Comment:
No points lost for this, but your test script shouldn't have ``.js`` at the end for some reason I'm not sure of, or else, it doesn't run properly. I changed it on my end so I could execute your code.